tools/state-viewer/src/util.rs

```diff
@@ -169,7 +170,7 @@ pub fn check_apply_block_result(
     let protocol_version = block.header().latest_protocol_version();
     let epoch_id = block.header().epoch_id();
     let shard_layout = epoch_manager.get_shard_layout(epoch_id).unwrap();
-    let shard_index = shard_layout.get_shard_index(shard_id);
+    let shard_index = shard_layout.get_shard_index(shard_id).map_err(Into::<EpochError>::into)?;
```

Comment:
Curiosity: is `map_err` necessary with `anyhow`?

Reply:
It seems like it, the compiler complains if I remove it.